src/content/guides/transferring-assets.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Transferring Assets
+description: Transfer Centrapay Assets
```

Comment:
@Wulingoh This description should be something that encourages users to click it - it will be shown in search results and link previews. Chat to the jesters if you need help wording this